packages/block-library/src/video/edit.js

```diff
@@ -232,11 +240,21 @@ class VideoEdit extends Component {
 											isDefault
 											onClick={ open }
 											ref={ this.posterImageButton }
+											aria-describedby={ `video-block__poster-image-description-${ instanceId }` }
```

Comment:
For simpler maintainability, this id could be assigned to a variable and used in both places.

Reply:
Also saves space :)

Reply:
Updated :)